CodeEdit/Features/NavigatorArea/SourceControlNavigator/Repository/Views/SourceControlNavigatorRepositoryView+outlineGroupData.swift

```diff
@@ -60,6 +60,15 @@ extension SourceControlNavigatorRepositoryView {
                             label: remote.name,
                             symbolImage: "vault",
                             imageColor: .teal,
+                            children: remote.branches?.map { branch in
```

Comment:
One small side effect of this is that I can use the "New Branch From" menu item on these, but the flow doesn't seem to actually work on remote branches. Is there a way to disable that menu item for these for now until that feature can be implemented?

![Screenshot 2024-07-11 at 10 05 10 PM](https://github.com/user-attachments/assets/1a07e654-3726-4a8f-9f4c-83c16552666c)

Reply:
> It looks like theres a way to make the default selected branch in this menu be incorrect. This was after making a new branch, switching to it, and pressing the push button. It should default to the selected branch but it seems the operationBranch wasn't updated correctly from the repository outline view.

This seems to work for me. Can you reproduce this with the latest?

Reply:
I implemented it. You can now switch to a remote branch and create a new branch from a remote branch. I also disabled rename branch for remote branches.